PROJECT

```diff
@@ -1,7 +1,7 @@
 version: "3"
-layout: 
+layout:
 - go.kubebuilder.io/v2
-projectName: flink-on-k8s-operator
+projectName: flink-operator
 domain: k8s.io
 repo: github.com/googlecloudplatform/flink-operator
```

Comment:
the domain should be `streamnative.io` now

and repo probably should point to our own fork